src/icons/GitHub.tsx

```diff
@@ -1,14 +0,0 @@
-/* eslint-disable max-len */
```

Comment:
Why this is being removed?

Reply:
I may be mistaken, but I couldn't find where it is being used